projects/packages/jetpack-mu-wpcom/src/features/help-center/class-help-center.php

```diff
@@ -83,12 +83,9 @@ public function enqueue_script( $variant, $dependencies, $version ) {
 		$script_dependencies = $dependencies ?? array();
 
 		if ( $variant === 'wp-admin' || $variant === 'wp-admin-disconnected' ) {
-			// Crazy high number to prevent Jetpack removing it
-			// https://github.com/Automattic/jetpack/blob/30213ee594cd06ca27199f73b2658236fda24622/projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php#L196.
```

Comment:
This comment shouldn't be relevant anymore, as we no longer load the masterbar class in https://github.com/Automattic/jetpack/pull/38419...